go.mod

```diff
@@ -4,10 +4,10 @@ go 1.16
 
 require (
 	github.com/aclements/go-moremath v0.0.0-20210112150236-f10218a38794
-	github.com/evergreen-ci/aviation v0.0.0-20211026175554-41a4410c650f
+	github.com/evergreen-ci/aviation v0.0.0-20211123195311-5ddfd75b3753
 	github.com/evergreen-ci/birch v0.0.0-20211025210128-7f3409c2b515
 	github.com/evergreen-ci/certdepot v0.0.0-20211109153348-d681ebe95b66
-	github.com/evergreen-ci/gimlet v0.0.0-20211029160936-5b64c7b33753
+	github.com/evergreen-ci/gimlet v0.0.0-20211119154230-7c9e13926e12
```

Comment:
do you know why all these other (!aviation) repos got updated? Was it on purpose?

Reply:
I think to match the version on aviation? I am not sure.